src/value.cpp

```diff
@@ -488,13 +488,11 @@ namespace Exiv2 {
         std::string c = comment;
         CharsetId charsetId = undefined;
         if (comment.length() > 8 && comment.substr(0, 8) == "charset=") {
-            std::string::size_type pos = comment.find_first_of(' ');
+            const std::string::size_type pos = comment.find_first_of(' ');
```

Comment:
if we are touching this line anyway, WDYT about:
```suggestion
            const auto pos = comment.find_first_of(' ');
```

Reply:
I think that might be a C++11 feature that we can't use on 0.27-maintenance.

Reply:
Sorry @kevinbackhouse :see_no_evil:  I forgot to take into account that this is of course only C++98.  
But that's also bad news for my other comments `back()` and `pop_back` are both C++11 features....

Sorry again!

Reply:
Good catch! I didn't realize that `back` and `pop_back` are C++11 too. I wonder why the build didn't break? I'll do a force push to revert these changes.